### PR TITLE
rustdesk-flutter: 1.2.3-unstable-2024-02-11 -> 1.2.3-2

### PR DESCRIPTION
**Pull request by dasJ**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/ru/rustdesk-flutter/package.nix [hunk 1]
```diff
@@ -41,30 +41,29 @@
...
+    rev = version;
+    hash = "sha256-aIGaj338eb8/l1NTgBwrLmx5VCpgGoYpx+p/ViCJgjQ=";
   };
 
   strictDeps = true;
 
   # Configure the Flutter/Dart build
   sourceRoot = "${src.name}/flutter";
-  # curl https://raw.githubusercontent.com/rustdesk/rustdesk/16db977fd81e14af62ec5ac7760a7661a5c24be8/flutter/pubspec.lock | yq
+  # curl https://raw.githubusercontent.com/rustdesk/rustdesk/1.2.3-1/flutter/pubspec.lock | yq
```
Comment (msanft) [inline pkgs/by-name/ru/rustdesk-flutter/package.nix:56]:
```suggestion
  # curl https://raw.githubusercontent.com/rustdesk/rustdesk/1.2.3-2/flutter/pubspec.lock | yq
```
Wouldn't this be the correct URL?

Comment (dasJ) [reply pkgs/by-name/ru/rustdesk-flutter/package.nix:56]: Thanks :)

**Comment by msanft** [comment]
Result of `nixpkgs-review pr 302519` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>rustdesk-flutter</li>
    <li>rustdesk-flutter.debug</li>
    <li>rustdesk-flutter.pubcache</li>
  </ul>
</details>

**Review by msanft — APPROVED** [review]

**Review by wineee — APPROVED** [review]

**Comment by wineee** [comment]
Build failed on `aarch64-linux`, but it is not the responsibility of this PR. It has never been successful.

**Pull request merged**